OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -85,7 +85,7 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager)
 				}
 
 				var team = item.Get<LabelWidget>("TEAM");
-				var teamNumber = pp.PlayerReference.Playable ? ((client == null) ? 0 : client.Team) : pp.PlayerReference.Team;
+				var teamNumber = pp.PlayerReference.Playable ? client.Team : pp.PlayerReference.Team;
```

Comment:
There's also some other checks for a null client you would need to remove, if you do believe the null check is redundant.